lib/services/abilities/message_abilities.ts

```diff
@@ -92,7 +99,9 @@ export function messageAbilities(staff: Staff) {
       sunshineMessageId: null,
       authorType: AuthorType.STAFF,
       authorId: staff.id,
-      metadata: opts?.metadata || {}
+      metadata: opts?.metadata || {},
+      customerDeliveryStatus: DeliveryStatus.UNSENT,
```

Comment:
These columns are weird in the sense that they only apply to staff to customer conversations and not staff to staff conversations.

It means that there will be a lot of non-customer conversations in the db with status 'unsent'

Reply:
We can add another status `NOT_APPLICABLE`, although for those messages `UNSENT` isn't wrong in the sense that it wasn't sent haha